mysqlcluster/mysqlcluster.go

```diff
@@ -114,12 +118,19 @@ func (c *MysqlCluster) GetSelectorLabels() labels.Set {
 
 // GetMySQLVersion returns the MySQL server version.
 func (c *MysqlCluster) GetMySQLVersion() string {
-	version := c.Spec.MysqlVersion
-	// lookup for an alias, usually this will solve 5.7 to 5.7.x
-	if v, ok := utils.MySQLTagsToSemVer[version]; ok {
+	var version string
+	// lookup for an alias, this will solve MySQL tags:
+	// 5.7 --> 5.7.x
+	// 8.0 --> 8.0.x
+	if v, ok := utils.MySQLTagsToSemVer[c.Spec.MysqlVersion]; ok {
 		version = v
+	} else {
+		errmsg := "Invalid mysql version option:" + c.Spec.MysqlVersion
+		log.Error(errors.New(errmsg), "currently we do not support mysql 5.6 or earlier version, default mysql version option should be 5.7 or 8.0")
+		return utils.InvalidMySQLVersion
 	}
 
+	// check if has specified image
```

Comment:
The first letter needs capitalization and the end of the sentence needs to be added `.`

Reply:
Done.